### PR TITLE
Add basic django forms encoder

### PR DESCRIPTION
**Pull request by stinovlas**

### CODE REVIEW

File: django_forms_encoder/encoder.py
Comment (thekarinka) [inline django_forms_encoder/encoder.py]: With newer version of schema there should be just `errors`

**Review by thekarinka — CHANGES_REQUESTED** [review]

File: django_forms_encoder/encoder.py
Comment (stinovlas) [reply django_forms_encoder/encoder.py]: Yeah, I'll update the code as soon as the schema is final.

Comment (stinovlas) [reply django_forms_encoder/encoder.py]: I pushed changes reflecting the new interface version.

**Review by thekarinka — APPROVED** [review]

**Comment by stinovlas** [comment]
Thanks for the approval, I'll wait for @ziima as well before I merge the branch.

File: django_forms_encoder/tests/test_encoder.py
Comment (ziima) [inline django_forms_encoder/tests/test_encoder.py]: I'd rather see this split into several tests, one for each feature of the encoder.

**Review by ziima — CHANGES_REQUESTED** [review]
I also miss the JSON schema for the output.

File: django_forms_encoder/tests/test_encoder.py
Comment (stinovlas) [reply django_forms_encoder/tests/test_encoder.py]: I tried to do that, but it leads to comparing long lists of inidividual properties, such as:

```python
self.assertEqual(
    [field['label'] for field in self.encoded_form['fields']],
    ['First name', 'Last name', 'email', 'age', 'is_admin', 'cv']
)
```

I feel that this is much less comprehensible. So, unless you have an idea how to split the tests and still keep them clear and easy to understand, I'll keep them as they are.

Comment (ziima) [reply django_forms_encoder/tests/test_encoder.py]: That is caused by the single large `TestForm`. I'd rather split that as well to separate small test forms in each test. Any change in the `TestForm` may now lead to several changes in unrelated tests, which complicates any subsequent changes.

Comment (stinovlas) [reply django_forms_encoder/tests/test_encoder.py]: Yeah, but that means having separate `Form` in each testcase, which makes the tests quite long and (again) less comprehensible.

Comment (stinovlas) [reply django_forms_encoder/tests/test_encoder.py]: I've completely overhauled encoder tests. I agree that it is more legible now, even if a little bit longer.

Comment (ziima) [reply django_forms_encoder/tests/test_encoder.py]: :+1:

**Comment by stinovlas** [comment]
I added the final encoded form JSON schema.

**Review by ziima — APPROVED** [review]

**Pull request merged**
